qubes-rpc/admin.vm.TerminalDispVM

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash --
+
+set -eu
+sock_name="$(mktemp -u XXXXXX)"
+if [[ "$sock_name" =~ [^a-zA-Z0-9] ]]; then
```

Comment:
What are you checking here? if `mktemp` worked? That should be handled by `set -e` already.